climpred/bootstrap.py

```diff
@@ -317,80 +381,199 @@ def _bootstrap_by_stacking(init_pm, control):
     return fake_uninit
 
 
-def _bootstrap_func(
```

Comment:
this part here got a bit messy, because I moved the stats functions to the end of the file